packages/react-kit/src/components/DateInput/DateInput.tsx

```diff
@@ -560,11 +570,12 @@ class RawDateInput extends React.Component<TDateInputFullProps, TDateInputState>
 			case ActiveSection.Month: {
 				if (this.secondInput) {
 					const newMonth = month.map(value => {
-						const monthValue = Number(`${value}${digit}`);
-						if (value < 1) {
+						const fixedValue = value + 1;
```

Comment:
you have `fxiedValue` here and `monthValue` above. This is:
1) not consistent
2) does not describe what stored in variable.

I suggest smth like `correctedMonth`